src/etc/one-context.d/loc-15-keepalived##vr

```diff
@@ -64,6 +64,12 @@ get_check_script() {
     echo $script
 }
 
+get_notify_script() {
```

Comment:
This function is unnecessary. The variable VROUTER_KEEPALIVED_NOTIFY_SCRIPT can be used directly inside the `gen_notify_script` function.